Makefile

```diff
@@ -21,3 +25,15 @@ show-quay-pull-urls: ## Show Quay.io pull for all images in repo
 update-fluentd-gemfile.lock: ## Update the fluentd Gemfile.lock file
 	docker run -v "$$PWD/fluentd/include:/docker-share" --workdir=/docker-share --rm -ti ruby bundle update
 
+.PHONY: build
+build:
+	docker-compose build ap-$(image_name)
+
+.PHONY: test
+test: export ASTRO_IMAGE_NAME = ap-$(image_name)
+test: export ASTRO_IMAGE_TAG = $(image_tag)
+test: export ASTRO_IMAGE_TEST_CONFIG_PATH = $(image_test_config)
+
+test:
```

Comment:
Need to add help text here so that `make help` works for this target.

Reply:
Done

Reply:
TIL you can do `make target var=val` and not just `var=val make target`